docs/topics/Gateway.md

```diff
@@ -791,6 +823,31 @@ A user's presence is their current state on a guild. This event is sent when a u
 }
 ```
 
+###### Example Game with Rich Presence
+
+```json
+{
+	"name": "Rocket League",
+	"type": 0,
+	"application_id": "379286085710381999",
+	"state": "In a Match",
+	"details": "Ranked Duos: 2-1",
+	"timestamps": {
+		"start": 1511200066
+	},
+	"party": {
+		"id": "35137100553872123",
```

Comment:
this is not guaranteed to be a snowflake

Reply:
I didn't write it down with type snowflake either